Source/part.f90

```diff
@@ -1649,13 +1649,11 @@ SUBROUTINE MOVE_IN_GAS
       LP%RE  = RHO_G*QREL*2._EB*R_D/MU_AIR
       C_DRAG = DRAG(LP%RE,LPC%DRAG_LAW)
 
-      ! Drag reduction model for liquid droplets
+      ! Drag reduction model for liquid droplets and solid particles
 
       WAKE_VEL=1.0_EB
-      IF (LPC%LIQUID_DROPLET) THEN
-         DROP_VOL_FRAC = MIN(1._EB,AVG_DROP_DEN(IIG_OLD,JJG_OLD,KKG_OLD,LPC%ARRAY_INDEX)/LPC%DENSITY)
-         IF (DROP_VOL_FRAC > LPC%DENSE_VOLUME_FRACTION) CALL WAKE_REDUCTION(DROP_VOL_FRAC,LP%RE,C_DRAG,WAKE_VEL)
-      ENDIF
+      DROP_VOL_FRAC = MIN(1._EB,AVG_DROP_DEN(IIG_OLD,JJG_OLD,KKG_OLD,LPC%ARRAY_INDEX)/LPC%DENSITY)
+      IF (DROP_VOL_FRAC > LPC%DENSE_VOLUME_FRACTION) CALL WAKE_REDUCTION(DROP_VOL_FRAC,LP%RE,C_DRAG,WAKE_VEL)
```

Comment:
@btrettel Are you sure that `LPC%DENSE_VOLUME_FRACTION` is defined for `.not.LPC%LIQUID_DROPLET`?

Reply:
I believe yes, as DENSE_VOLUME_FRACTION is on the PART line and doesn't seem to have anything specifically to do with liquid droplets or solid particles. I'll look at what the FDS code does, however.